test/jdk/jdk/modules/incubator/ImageModules.java

```diff
@@ -24,14 +24,15 @@
 /*
  * @test
  * @bug 8170859
- * @summary Basic test for incubator modules in jmods and images
+ * @requires vm.flagless
```

Comment:
Also, if I understand correctly, this test is marked as flagless because it uses `executeCommand` directly? (it doesn't seem to use `createLimtiedTestJavaProcessBuilder`)

Reply:
It launches `java` from the custom image created by jlink.   `createLimtiedTestJavaProcessBuilder` and `createTestJavaProcessBuilder` launches `java` from `test.jdk`.